packages/botonic-plugin-contentful/src/render/botonic-converter.ts

```diff
@@ -149,6 +149,6 @@ export class BotonicMsgConverter {
     if (this.options.replaceEmptyStringsWith == undefined) {
       return str
     }
-    return str || ' '
+    return str || this.options.replaceEmptyStringsWith
```

Comment:
could be useful to use `??` here also?

Reply:
str will never be undefined, and when it's "" I want this.options.replaceEmptyStringsWith, so it wouldn't work